packages/kit/package.json

```diff
@@ -26,6 +26,7 @@
 		"kleur": "^4.1.5",
 		"magic-string": "^0.30.5",
 		"mrmime": "^2.0.0",
+		"pathe": "^1.1.2",
```

Comment:
We try really hard not to add new dependencies. Let's find a way to do this without adding one

Reply:
This is to use in place of `node:path` for compatibility with other runtimes. We're only using the `resolve` function though so we could probably create our own with the same behaviour.